### ISSUE TITLE
HIDPI support broken on Macos builds

### ISSUE THREAD

**Issue by lpkruger** [Genymobile/scrcpy#829]
The issue is straightforward - screen.c does not #include "config.h" as it should so it never picks up the proper #ifdef from the Meson config.

It worked in 1.8 perhaps by accident because screen.c includes video_buffer.h which used to include config.h.  This was removed in 
https://github.com/Genymobile/scrcpy/commit/ebccb9f6cc111e8acfbe10d656cac5c1f1b744a0

with the result that screen.c no longer picks up the Meson preprocessor defines.

**Comment by lpkruger** [comment]
BTW - grepping for "config.h" shows there are quite a few source files that do not have this include.    Since it is difficult to immediately know which file depends on Meson configs and which one does not, perhaps they should all include config.h ?

**Comment by lpkruger** [comment]
here is a tested fix for the HIDPI issue

```
Author: Louis Kruger <louisk@gmail.com>
Date:   Sat Sep 28 18:54:08 2019 -0400

    include config.h

diff --git a/app/src/screen.c b/app/src/screen.c
index 18d24dd..e34bcf4 100644
--- a/app/src/screen.c
+++ b/app/src/screen.c
@@ -3,6 +3,7 @@
 #include <string.h>
 #include <SDL2/SDL.h>

+#include "config.h"
 #include "common.h"
 #include "compat.h"
 #include "icon.xpm"
```

**Comment by rom1v** [comment]
> here is a tested fix for the HIDPI issue

Do you mean #15? :tada:

> screen.c does not #include "config.h"

:scream:

> Since it is difficult to immediately know which file depends on Meson configs and which one does not, perhaps they should all include config.h ?

:+1:

**Comment by rom1v** [comment]
129dabcfa466d0d76d2f2b2715e48d649d80a4ba 1f8ba1ca79f4915f3a7b3c5199258904e3192445

**Comment by lpkruger** [comment]
Sorry I do not know whether this fixed #15 or not.  If I get a free minute I could look into that though.
